lumigator/infra/mzai/helm/lumigator/values.yaml

```diff
@@ -10,8 +10,18 @@ postgresDb: ""
 postgresHost: ""
 postgresPort: ""
 postgresUser: ""
+
+# The password of the postgres database to connect to.
+# A secret will be created for you.
+# Set this OR 'existingPostgresPasswordSecretName'
 postgresPassword: ""
 
+# Use an existing secret, set this value OR use 'postgresPassword'
+existingPostgresPasswordSecretName: ""
```

Comment:
We should clarify that the user has to use only of the two options (provide the key or the secret), like we did with the [keys](https://github.com/mozilla-ai/lumigator/blob/main/lumigator/infra/mzai/helm/lumigator/README.md#mistral-api-key-management)